app/src/main/java/co/nimblehq/alpine/sample/CameraCaptureActivity.kt

```diff
@@ -0,0 +1,98 @@
+package co.nimblehq.alpine.sample
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import androidx.activity.ComponentActivity
+import androidx.camera.core.*
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.core.content.ContextCompat
+import co.nimblehq.alpine.databinding.ActivityCameraCaptureBinding
+import co.nimblehq.alpine.sample.extension.getStatusBarHeight
+import co.nimblehq.alpine.sample.extension.setLightStatusBar
+
+open class CameraCaptureActivity : ComponentActivity() {
+
+    companion object {
```

Comment:
Don't companion objects typically go at the bottom of a class? 😄

Reply:
No biggie, IMHO. But according to [Jetbrain's convention](https://kotlinlang.org/docs/coding-conventions.html#class-layout), they recommend placing the `companion object`s at the bottom.